src/app/services/baw-api/baw-api.service.ts

```diff
@@ -122,10 +122,10 @@ export abstract class BawApiService<Model extends AbstractModel> {
   /**
    * Retrieve user details from session cookie. Null if no user exists.
    */
-  public getLocalUser(): SessionUser | null {
+  public getLocalUser(): SessionUser | undefined {
```

Comment:
Comment out of date?

Possibility for null still exists down below, but then type signature out of date?